### PR TITLE
Moves end round info to its own panel

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request

You know how when you play regular SS13 and you finish a round a panel pops out in the middle of your screen with the round info? This is that but for TGMC.
I mostly used code from TG for this, but there's a few minor areas that received minor rewrites.
It should display all the info and stats that are normally displayed at the end of a round, I've verified this locally but it needs a TM to be entirely sure.

~~Pictures soon.~~
A week later apparently
endround

## Why It's Good For The Game

Have you ever had a round end, the stats and results get pushed to chat and then it scrolls so fast you have to scroll up for five minutes to find out who won, fighting against the tide of new chat messages? This should put an end to that for good, the panel is placed in the middle of the screen and there's a handy button for reopening it.

To my knowledge, CM and TGMC are the only codebases that do not push a notification to the screen on round end, and I intend to change this. ~~I also need this for future features I'm working on~~

## Changelog
:cl:
qol: Round end info is now displayed in its own panel instead of in chat
/:cl:

### CODE REVIEW

File: code/__DEFINES/admin.dm [hunk 1]
```diff
@@ -75,6 +75,8 @@
 #define AHELP_RESOLVED 3
 
 #define ROUNDSTART_LOGOUT_REPORT_TIME 10 MINUTES //Amount of time (in deciseconds) after the rounds starts, that the player disconnect report is issued.
+#define ROUNDEND_EORG_DELAY 10 SECONDS //Amount of time (in deciseconds) after the rounds starts, that the player disconnect report is issued.
```
Comment (TiviPlus) [inline code/__DEFINES/admin.dm:78]: Comment

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Whats with the map changes and these
image
Also pics soontm

**Comment by RipGrayson** [comment]
> Whats with the map changes and these image Also pics soontm

Map changes because there was runtimes during testing related to a bad object in a map. It was a one line fix so I just lumped it in. I've also made some random smooth group adjustments because Lumi discovered some bugs and the changes weren't big enough to justify their own PR.

**Comment by RipGrayson** [comment]
Based off feedback given on Discord, I have opted to remove the "Thus ends the story" section from the panel. It will now push to chat like it does currently.

**Comment by RipGrayson** [comment]
PR is borked somehow, I'll reopen this on a new branch

**Pull request closed without merging**